src/modules/claim/claim.service.ts

```diff
@@ -51,10 +51,16 @@ export class ClaimService {
   }
 
   private initializeExchangeListener() {
-    this.nats.connection.subscribe(`*.${NATS_EXCHANGE_TOPIC}`, async data => {
-      this.logger.debug(`Got message ${data}`);
-      await this.claimQueue.add('save', data);
-    });
+    this.nats.connection.subscribe(
+      `*.${NATS_EXCHANGE_TOPIC}`,
+      {
+        queue: 'iam_cache_server_exchange_topic',
```

Comment:
@hejkerooo where is this queue name from? Is this configured in NATS?

Reply:
@hejkerooo, thanks for the change btw, I didn't know about NATS queues, this is pretty awesome